src/app/modules/product/component/category/category.component.html

```diff
@@ -0,0 +1,81 @@
+<body>
+  <header>
+    <nav class="navbar navbar-expand-lg bg-primary" data-bs-theme="dark">
+      <div class="container-fluid navbar-content">
+        <a class="navbar-brand" href="#">DW-FrontEnd</a>
+        <button class="navbar-toggler" type="button" data-bs-toggle="collapse" data-bs-target="#navbarColor01"
+          aria-controls="navbarColor01" aria-expanded="false" aria-label="Toggle navigation">
+          <span class="navbar-toggler-icon"></span>
+        </button>
+        <div class="collapse navbar-collapse" id="navbarColor01">
+          <ul class="navbar-nav me-auto">
+            <li class="nav-item">
+              <a class="nav-link active" href="#">Category
+                <span class="visually-hidden">(current)</span>
+              </a>
+            </li>
+            <li class="nav-item">
+              <a class="nav-link" href="#">Home</a>
+            </li>
+            <li class="nav-item">
+              <a class="nav-link" href="#">Features</a>
+            </li>
+            <li class="nav-item">
+              <a class="nav-link" href="#">About</a>
+            </li>
+            <li class="nav-item dropdown">
+              <a class="nav-link dropdown-toggle" data-bs-toggle="dropdown" href="#" role="button" aria-haspopup="true"
+                aria-expanded="false">Dropdown</a>
+              <div class="dropdown-menu">
+                <a class="dropdown-item" href="#">Action</a>
+                <a class="dropdown-item" href="#">Another action</a>
+                <a class="dropdown-item" href="#">Something else here</a>
+                <div class="dropdown-divider"></div>
+                <a class="dropdown-item" href="#">Separated link</a>
+              </div>
+            </li>
+          </ul>
+          <form class="d-flex">
+            <input class="form-control me-sm-2" type="search" placeholder="Search">
+            <button class="btn btn-secondary my-2 my-sm-0" type="submit">Search</button>
+          </form>
+        </div>
+      </div>
+    </nav>
+  </header>
+  <main>
+    <div class="body-content">
+      <div class="card">
+        <div class="card-header">
+          <h4>Categorías</h4>
+        </div>
+        <div class="card-body">
+          <table class="table table-striped">
+            <thead>
+              <tr class="table-header">
+                <th scope="col">Category Id</th>
+                <th scope="col">Category</th>
+                <th scope="col">Tag</th>
+                <th scope="col">Status</th>
+              </tr>
+            </thead>
+            <tbody>
+              @for(category of categories; track category.category_id; let i = $index ) {
+              <tr>
+                <td>{{ category.category_id }}</td>
+                <td>{{ category.category }}</td>
+                <td>{{ category.tag }}</td>
+                <td>{{ category.status }}</td>
```

Comment:
Como sugerencia, podrían utilizar la directiva `ngIf` para colocar a manera de texto el status de la categoría, asignando 1 a _"Activo"_ y 0 a _"Inactivo"_ como sigue:
```suggestion
                        <td> @if (category.status) {
                            Activo
                            } @else {
                            Inactivo
                            }
                        </td>
```